src/js/datetimepicker.js

```diff
@@ -132,6 +195,9 @@
         replace: true,
         link: function link(scope, element, attrs, ngModelController) {
 
+          scope.showCurrent = attrs.showCurrent ? scope.$parent.$eval(attrs.showCurrent)
+                                                : ['year','month','day','hour','minute'];
```

Comment:
I don't that code belongs in the dateObject - just an attribute of today. Is there some reason something more is needed?